test/data/manifests/fedora-coreos-container-s390x.mpp.yaml

```diff
@@ -0,0 +1,450 @@
+version: '2'
+mpp-vars:
+  arch: s390x
+  disk_size_gb: 10
+  boot_size_mb: 384
+  root_size_mb: 2048
+  sector_size: 512
+  four_k_sector_size: 4096
+  # The name to use for deployment stateroot
+  osname: fedora-coreos
+  # The container image ref to follow for updates
+  container_imgref: ostree-remote-registry:fedora:quay.io/fedora/fedora-coreos:stable
+  # Filesystem UUID and label definitions. These UUIDs
+  # are looked for on boot and if found replaced with
+  # a new random UUID to make each install unique.
+  boot_fs_uuid: 96d15588-3596-4b3c-adca-a2ff7279ea63
+  boot_fs_label: boot
+  root_fs_uuid: 910678ff-f77e-4a7d-8d53-86f2ac47a823
+  root_fs_label: root
+mpp-define-images:
+  - id: image
+    size:
+      mpp-format-string: "{disk_size_gb * 1024 * 1024 * 1024}"
+    table:
+      uuid: 00000000-0000-4000-a000-000000000001
+      label: gpt
+      partitions:
+        - name: boot
+          type: 0FC63DAF-8483-4772-8E79-3D69D8477DE4
+          partnum: 3
+          size:
+            mpp-format-int: "{boot_size_mb * 1024 * 1024 / sector_size}"
+        - name: root
+          type: 0FC63DAF-8483-4772-8E79-3D69D8477DE4
+          partnum: 4
+          size:
+            mpp-format-int: "{root_size_mb * 1024 * 1024 / sector_size}"
+  - id: image4k
+    sector_size:
+        mpp-format-int: "{four_k_sector_size}"
+    size:
+      mpp-format-string: "{disk_size_gb * 1024 * 1024 * 1024}"
+    table:
+      uuid: 00000000-0000-4000-a000-000000000001
+      label: gpt
+      partitions:
+        - name: boot
+          partnum: 3
+          type: 0FC63DAF-8483-4772-8E79-3D69D8477DE4
+          size:
+            mpp-format-int: "{boot_size_mb * 1024 * 1024 / four_k_sector_size}"
+        - name: root
+          partnum: 4
+          type: 0FC63DAF-8483-4772-8E79-3D69D8477DE4
+          size:
+            mpp-format-int: "{root_size_mb * 1024 * 1024 / four_k_sector_size}"
+pipelines:
+  - mpp-import-pipelines:
+      path: fedora-vars-s390x.ipp.yaml
+  - mpp-import-pipeline:
+      path: fedora-build-v2-s390x.ipp.yaml
+      id: build
+    runner:
+      mpp-format-string: org.osbuild.fedora{release}
+  - name: tree
+    build: name:build
+    source-epoch: 1659397331
+    stages:
+      - type: org.osbuild.ostree.init-fs
+      - type: org.osbuild.ostree.os-init
+        options:
+          osname:
+            mpp-format-string: '{osname}'
+      - type: org.osbuild.ostree.config
+        options:
+          repo: /ostree/repo
+          config:
+            sysroot:
+              readonly: true
+              bootloader: none
```

Comment:
Here it should be `zipl`, but than build fails. Firstboot and subsequent boot work, but any change to `sysroot` (like `rpm-ostree kargs --append=foo`) break the system. It's possible to workaround that after ignition finished with manual call to `ostree` . 
Anyhow debugging now

Reply:
Fixed now